src/Components/AddItem/AddItem.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import Select from 'react-select';
+import axios from 'axios';
+import { BsCheck2Circle } from 'react-icons/bs'
+import { AiOutlineClose } from 'react-icons/ai'
+import { FaExclamationCircle } from 'react-icons/fa'
+import './AddItem.css'
+
+function AddItem() {
+
+    const[inputs, setInputs] = useState({
+        product: '',
+        department: '',
+        packeging: ''
+    })
+
+    const departments = [
+        { value: 'Vegetables', label: 'Vegetables' },
+        { value: 'Fruits', label: 'Fruits' },
+        { value: 'Dairy', label: 'Dairy' },
+        { value: 'Frozen', label: 'Frozen' },
+        { value: 'Meats', label: 'Meats' },
+        { value: 'Deli', label: 'Deli' },
+        { value: 'Canned Food', label: 'Canned Food' },
+        { value: 'Beauty', label: 'Beauty' },
+        { value: 'Cleaning', label: 'Cleaning' },
+        { value: 'Spices', label: 'Spices' },
+        { value: 'Cereals', label: 'Cereals' },
+        { value: 'Drinks', label: 'Drinks' },
+        { value: 'General', label: 'General' },
+    ];
+
+    const packeging = [
+        { value: 'Individual', label: 'Individual' },
+        { value: 'KG', label: 'KG' },
+        { value: 'Gram', label: 'Gram' },
+        { value: 'Pack', label: 'Pack' }
+    ];
+    
+    const [selectedDepartment, setSelectedDepartment] = useState(null);
+    const [selectedPackeging, setSelectedPackaging] = useState(null);
+    const [inserted, setInserted] = useState(false);
+    const [notInserted, setNotInserted] = useState(false);
```

Comment:
What? why do you need 2 inverted consts? just use `!inserted`

Reply:
the name is confusing, it's not inverted consts
"inserted" means item added successfully into database
"notInserted" means something went wrong and the item hasn't been added